src/Synth/OscilGen.cpp

```diff
@@ -1327,9 +1328,9 @@ void OscilGen::add2XML(XMLwrapper& xml)
 
         xml.beginbranch("BASE_FUNCTION");
         for(int i = 1; i < synth.oscilsize / 2; ++i) {
-            float xc = basefuncFFTfreqs[i].real();
-            float xs = basefuncFFTfreqs[i].imag();
-            if((fabs(xs) > 1e-6f) || (fabs(xc) > 1e-6f)) {
+            float xc = (float)basefuncFFTfreqs[i].real();
+            float xs = (float)basefuncFFTfreqs[i].imag();
+            if((fabsf(xs) > 1e-6f) || (fabsf(xc) > 1e-6f)) {
```

Comment:
All float casts and func()->funcf() changes not otherwise noted in this file look correct.